modules/auxiliary/scanner/http/http_version.rb

```diff
@@ -26,14 +26,25 @@ def initialize
     register_wmap_options({
         'OrderID' => 0,
         'Require' => {},
-      })
+    })
+
+    register_options([
```

Comment:
This is probably better off as `register_advanced_options` , or at least for `HTTP_METHOD` and `PROTOCOL`, which will rarely need overriding.